src/Alert.php

```diff
@@ -82,7 +134,7 @@ public function body(string $value): self
      *
      * @return self
      */
-    public function closeButton(array $value): self
+    public function closeButton(?array $value): self
```

Comment:
Because you need null here, if you have `withoutButton()`, you can even remove the `null` and pass the `empty array`, you will have less checks.

Reply:
Sometimes not needed styling close button - it can be just text "Close" instead of X or standart button if it outside

Reply:
And why do you need the null?

Reply:
> And why do you need the null?

Semantic - null = not set/exists ))

Reply:
Well yes it is acceptable, although personally an empty array does the same thing, and you don't need to check if it is null.